src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/ViewModels/DataSetVersionChangesViewModel.cs

```diff
@@ -19,11 +26,17 @@ public record DataSetVersionChangesViewModel
     /// Any minor changes that were made to the data set.
     /// </summary>
     public required ChangeSetViewModel MinorChanges { get; init; }
+}
 
+/// <summary>
+/// A set of changes made to a data set version, can include any patch change sets associated with the data set version if it has patch version history.
```

Comment:
I'd update `data set version, can include` to `data set version that can optionally include`